nprapps/elections-bots
# GitHub actions

## Describe your changes
I created GitHub Actions to automate it. It still runs on push, but I will have to update it when it reaches the final stage to run on a corn job. 

## Issue ticket number and link
Ongoing work for [165](https://github.com/nprapps/elections24-primaries/issues/165)

## Testing Steps
To test if Github actions work as expected, 

- Go to [Actions](https://github.com/nprapps/elections-bots/actions) tab
- Delete data from this [Google sheet](https://docs.google.com/spreadsheets/d/11NMkh8GqUYRCHKMKuPu9bGsS7G1sSbMaDk0YaLxNdSI/edit#gid=0)
- Rerun the latest green action
- Check the [Google sheet](https://docs.google.com/spreadsheets/d/11NMkh8GqUYRCHKMKuPu9bGsS7G1sSbMaDk0YaLxNdSI/edit#gid=0). It should be updated with the latest data
- Check #elex-bots-test channel for a generic message.
